examples/tests/sdk_tests/dynamodb-table/global-secondary-index.test.w

```diff
@@ -0,0 +1,93 @@
+bring cloud;
+bring ex;
+bring util;
+
+let api = new cloud.Api();
+
+let table = new ex.DynamodbTable(
+    name: "blog",
+    attributeDefinitions: {
+        type: "S",
+        id: "S",
+        createdAt: "N"
+    },
+    hashKey: "type",
+    rangeKey: "id",
+    globalSecondaryIndex: [{
+        name: "CreatedAtIndex",
+        hashKey: "type",
+        rangeKey: "createdAt",
+        projectionType: "ALL"
+    }],
+);
+
+// Only Hash Key
+let table2 = new ex.DynamodbTable(
+    name: "blog2",
+    attributeDefinitions: {
+        type: "S",
+        id: "S",
+    },
+    hashKey: "type",
+    rangeKey: "id",
+    globalSecondaryIndex: [{
+        name: "TypeIndex",
+        hashKey: "type",
+        projectionType: "ALL"
+    }],
+) as "blog2";
+
+test "Global secondary index" {
+    for i in 1..11 {
+        let var id = util.nanoid();
+
+        // Make the first item entered last, and the last item first
+        if i == 1 {
+            id = "0" + id;
+        } elif i == 10 {
+            id = "z" + id;
+        }
```

Comment:
Doesn't 0 appear before z? The first one will be entered first, and the last one last

Reply:
I changed the test, I entered the records manually to avoid any unexpected situation.